.github/workflows/release-stable.yml

```diff
@@ -54,33 +26,55 @@ jobs:
           distribution: 'zulu'
           java-package: 'jdk+fx'
 
-      - name: Validate Gradle wrapper
-        uses: gradle/wrapper-validation-action@56b90f209b02bf6d1deae490e9ef18b21a389cd4
-
       - name: Gradle build
         uses: gradle/gradle-build-action@842c587ad8aa4c68eeba24c396e15af4c2e9f30a
         with:
           arguments: build
 
+      - name: Remove Snapshot Suffix
+        id: removeSnap
```

Comment:
> Snap is a software packaging and deployment system developed by Canonical for operating systems that use the Linux kernel and the systemd init system.

Better call this "removeSnapshot" to avoid any misunderstandings.